File: java/BillsApi/src/main/java/com/apilangarena/javabillsapi/presentation/dto/CreateBillResponse.java
package com.apilangarena.javabillsapi.presentation.dto;

import java.math.BigDecimal;
import java.time.LocalDate;

public record CreateBillResponse(
    Long id,
    String billNumber,
    LocalDate issuedAt,
    BigDecimal subtotal,
    BigDecimal tax,
    BigDecimal total,
    String currency
) {
}

File: kotlin/BillsApi/src/main/kotlin/com/apilangarena/billsapi/presentation/dto/CreateBillResponse.kt
package com.apilangarena.billsapi.presentation.dto

import java.math.BigDecimal
import java.time.LocalDate

data class CreateBillResponse(
    val id: Long,
    val billNumber: String,
    val issuedAt: LocalDate,
    val subtotal: BigDecimal,
    val tax: BigDecimal,
    val total: BigDecimal,
    val currency: String,
)
